functions/src/pdfBuilder/generateDonationCertificatePDF.ts

```diff
@@ -0,0 +1,156 @@
+import { connectStorageEmulator} from 'firebase/storage';
+import { getStorage, ref, uploadBytes, getDownloadURL } from "firebase/storage";
+import { doc, setDoc } from "firebase/firestore";
+import * as config from '../config';
+
+let STORAGE_EMULATOR_REQUIRED = true;
+
+export const createAndUploadDonationCertificate = async (data: any, app:any, firestore: any) => {
+
+	const storage = getStorage(app); 
```

Comment:
I would suggest that we have a separate file `useStorageAdmin.ts` which exposes the admin storage (see explanation between the difference in the security rule part). Then we have a consistent setup for storage and firestore. Let me create a separate PR for this 👍

Reply:
https://github.com/socialincome-san/public/pull/161